pkg/agent/proxy/proxier.go

```diff
@@ -0,0 +1,270 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proxy
+
+import (
+	"sync"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+	"github.com/vmware-tanzu/antrea/pkg/agent/proxy/types"
+	"github.com/vmware-tanzu/antrea/pkg/agent/proxy/upstream"
+	"github.com/vmware-tanzu/antrea/pkg/agent/proxy/upstream/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/querier"
+	binding "github.com/vmware-tanzu/antrea/pkg/ovs/openflow"
+)
+
+const (
+	resyncPeriod  = time.Minute
+	componentName = "antrea-agent-proxy"
+)
+
+// TODO: Add metrics
+type Instance struct {
```

Comment:
Feel the original name Proxier is better than Instance as Instance doesn't have any specific mean to what it does.
https://golang.org/doc/effective_go.html#names

Reply:
Done.